components/dash-core-components/src/utils/LazyLoader/mathjax.js

```diff
@@ -0,0 +1,9 @@
+export default (mathjax) => Promise.resolve(window.MathJax || (
+    mathjax === false ?
+        undefined :
+        import(/* webpackChunkName: "mathjax" */ 'mathjax/es5/tex-svg.js').then(() => {
+            window.MathJax.config.startup.typeset = false;
```

Comment:
Ah shoot, this `typeset = false` isn't included in the chunk, so in eager mode we may end up going ahead with the initial typesetting. Most of the time this shouldn't be a problem because the page won't have any content on it at that point, but in dash-embedded it could, and that's now perhaps the major reason people would use `eager_loading=True`.

I think what we may need to do is make a separate file that's just:
```js
import 'mathjax/es5/tex-svg.js';

window.MathJax.config.startup.typeset = false;
```
And then in this file we `import()` THAT file as the `mathjax` chunk.